dask/array/tests/test_fft.py

```diff
@@ -108,3 +113,41 @@ def test_fft_consistent_names():
     assert same_keys(fft(darr, 5), fft(darr, 5))
     assert same_keys(fft(darr2, 5, axis=0), fft(darr2, 5, axis=0))
     assert not same_keys(fft(darr, 5), fft(darr, 13))
+
+
+def test_wrap_numpy():
```

Comment:
> Is the PyFFTW library consistent with the numpy interface?

Yes, they provide a numpy interface that they try to keep up-to-date. Here's the [docs]( https://hgomersall.github.io/pyFFTW/pyfftw/interfaces/numpy_fft.html#module-pyfftw.interfaces.numpy_fft ) for more details.

Was debating adding optional tests for it, but was unclear if we want to go that route. One thing to consider is FFTW is GPL (or some license fee to remove GPL) while pyFFTW is BSD. Maybe it would be ok to an optional test. I'll defer to you though.